v20.1/cockroachcloud-cluster-management.md

```diff
@@ -32,11 +28,26 @@ For each node, the page displays the node's `Name`, nested under its region.
 
 From the **Cluster Overview** page, you can connect to your cluster. For more information, see [Connect to Your CockroachCloud Cluster](cockroachcloud-connect-to-your-cluster.html).
 
-## Manage cluster configuration
+## Add or remove nodes from a cluster
+
+You can add or remove nodes from your cluster through the Console.
+
+{{site.data.alerts.callout_info}}
+At this time, we do not support scaling up a single-node cluster or scaling down to a single-node cluster.
```

Comment:
Can we add "if you want to scale down to 3 nodes or scale up your 1 node cluster, please open a support ticket at support.cockroachlabs.com.

Reply:
Updated with the text we discussed on Slack